src/main/java/ua/lviv/iot/parkingServer/service/ParkingSpotService.kt

```diff
@@ -1,50 +1,27 @@
 package ua.lviv.iot.parkingServer.service
 
-import jakarta.annotation.PostConstruct
-import jakarta.annotation.PreDestroy
-import java.util.concurrent.ConcurrentHashMap
-import java.util.concurrent.atomic.AtomicLong
 import org.springframework.stereotype.Service
-import ua.lviv.iot.parkingServer.datastorage.ParkingSpotFileStore
+import ua.lviv.iot.parkingServer.repository.ParkingSpotMongoRepository
 import ua.lviv.iot.parkingServer.exception.EntityNotFoundException
 import ua.lviv.iot.parkingServer.model.ParkingSpot
 import ua.lviv.iot.parkingServer.service.interfaces.ParkingSpotServiceInterface
 
 @Service
-class ParkingSpotService(private val parkingSpotFileStore: ParkingSpotFileStore) : ParkingSpotServiceInterface {
-    private val parkingSpots: MutableMap<Long, ParkingSpot> = ConcurrentHashMap()
-    private var id: AtomicLong = AtomicLong(1L)
+class ParkingSpotService(private val parkingSpotMongoRepository: ParkingSpotMongoRepository) :
+    ParkingSpotServiceInterface {
+    override fun findAllEntities(): List<ParkingSpot> = parkingSpotMongoRepository.findAll()
```

Comment:
maybe it would be look better if we add `\n`:

```kotlin
ParkingSpotServiceInterface {

override fun findAllEntities(): List<ParkingSpot> = parkingSpotMongoRepository.findAll()
```

Moreover, let's change code style:

```kotlin
class ParkingSpotService(
    private val parkingSpotMongoRepository: ParkingSpotMongoRepository
) : ParkingSpotServiceInterface {
```

Reply:
✅